dspace-spring-rest/src/main/java/org/dspace/app/rest/model/hateoas/RelationshipTypeResourceWrapper.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.model.hateoas;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import org.dspace.app.rest.model.RelationshipTypeRest;
+import org.dspace.app.rest.model.RelationshipTypeRestWrapper;
+import org.dspace.app.rest.utils.Utils;
+
+public class RelationshipTypeResourceWrapper extends HALResource<RelationshipTypeRestWrapper> {
```

Comment:
Javadocs for class and constructor (see below)

Reply:
The JavaDocs have been included now